coriolis_openstack_utils/subnets.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+
+def get_subnet(openstack_client, name_or_id):
+    return openstack_client.neutron.find_resource('subnet', name_or_id)
```

Comment:
Make it also take as param the network ID and filter by that. Raise only of the are multiple subnets named the same in the same network.